src/app/projects/projects-list/projects-list.component.ts

```diff
@@ -174,5 +182,16 @@ export class ProjectsListComponent implements OnInit {
       ...this.filters.getValue(),
       searchVal: $search,
     });
+
+    const LIMIT = 5;
+
+    window['dataLayer'].push({
```

Comment:
I would put all the google analytics stuff in its own Angular Service.

Reply:
Good suggestion. I was just waiting for the result of the conversation with Sandy.
After having the discussion around VF's GA component we decided to go with our own code as the VF component can't be used as we would like to.
So, I am going to do the service refactor now.

Reply:
@amnonkhen  I extracted this code to a service and pushed it. Could you review it again please? Tx